data/json/monsters/zed_tentacle.json

```diff
@@ -0,0 +1,201 @@
+[
+  {
+    "id": "mon_zombie_armed",
+    "type": "MONSTER",
+    "name": { "str": "armed zombie" },
+    "description": "This zombie has extremely elongated arms, tearing through the tracksuit it still wears. While the arms are menacingly long, they seem quite thin and floppy, as if they didn't have bones anymore.",
```

Comment:
replace "tracksuit" with just "clothing" or something?

Reply:
Also needs a double space before second sentence, as in "...wears.  While"

Reply:
Done!